utilities/path_follow.py

```diff
@@ -0,0 +1,89 @@
+from typing import List
+
+from wpilib import controller
+from wpilib import trajectory
+from wpilib import Timer
+from wpilib.trajectory import constraint
+from wpilib.geometry import Pose2d, Translation2d
+
+from components.chassis import Chassis
+
+
+class Path:
+    start: Pose2d
+    waypoints: List[Translation2d]
+    end: Pose2d
+    reversed: bool
+
+    def __init__(self, points, reversed) -> None:
+        self.start, *self.waypoints, self.end = points
```

Comment:
Since `points` isn't a homogeneous list, would it be better to just pass in `start` and `end` separately? (At which point it'd be worth making this a dataclass.)